configuration/elastic_search_mappings/intel_owl_bi.json

```diff
@@ -0,0 +1,42 @@
+// PUT intel_owl_bi
```

Comment:
if that is automatically uploaded, does it work with the `//`?

Reply:
json load ignore that line, but removed anyway